app/assets/stylesheets/application.css

```diff
@@ -14,6 +14,15 @@
  *= require_self
  */
 
+<<<<<<< HEAD
```

Comment:
**issue:** we must remove the merge conflict marks
```
<<<<<<< HEAD
=======
>>>>>>> main
```

Reply:
I missed this one 😬